### PR TITLE
Fix managed vars filters 

### LINKED ISSUE #2788: Errors in managed variants filters
While testing on stage I've realized that filters don't work. From the managed variants page (https://scout-stage.scilifelab.se/managed_variant) try for instance to filter variants by chromosome.

Genome build field in that form looks also funny: 

image

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2788

- Attempt at fixing the weird looking chromosome build on managed variants filter form 
- Added the code to be able to filter managed variants by chromosome coordinates
- Deactivate managed variants filter by chromosome build, since build can't be assigned when managed vars are created (yet)

**How to test**:
1. Install on clinical-db stage and try to filter managed variants with different parameters

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is ready for review and installed on clinical-db stage now

**Comment by dnil** [comment]
Hm, no, the size filters seem disabled now?

**Comment by northwestwitch** [comment]
> Hm, no, the size filters seem disabled now?

It WAS on stage

**Comment by dnil** [comment]
> > Hm, no, the size filters seem disabled now?
> 
> It WAS on stage

Sure, sure, but I also tested locally?

**Comment by northwestwitch** [comment]
> Sure, sure, but I also tested locally?

Ah OK, I'll check tomorrow!

**Comment by dnil** [comment]
I'll try it on stage real quick as well to be safe! 😅

**Comment by dnil** [comment]
But the chromosome filter looks like it is working fine!

File: scout/server/blueprints/managed_variants/controllers.py [hunk 1]
```diff
@@ -22,6 +22,23 @@
...
+
+    Args:
+        query_options(dict): managed variants optional params
+        request_form(ImmutableMultiDict): form submitted by user to filter managed variants
+    """
+    chrom = request_form.get("chromosome")
+    if chrom is None or chrom == "All":
+        return
+    query_options["chromosome"] = chrom
+    if request_form.get("position"):
```
Comment (dnil) [inline scout/server/blueprints/managed_variants/controllers.py:36]: Im guessing these would need to go before the return?

**Comment by northwestwitch** [comment]
@dnil I've changed so now the chromosome coordinates fields are disabled unless you select a single chromosome

File: scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html [hunk 2]
```diff
@@ -339,6 +342,20 @@ <h5>
     setCytobands(); // Reset cytobands select elements
   }
 
+  function set_coords_fields(){
```
Comment (dnil) [inline scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html:345]: 👍

File: scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html [hunk 3]
```diff
@@ -237,13 +237,13 @@ <h5>
         {{ wtf.form_field(filters_form.build, onchange="setChromosomes()") }}
       </div>
       <div class="col-2">
-        {{ wtf.form_field(filters_form.chromosome, onchange="setCytobands()") }}
+        {{ wtf.form_field(filters_form.chromosome, onchange="setCytobands();set_coords_fields()") }}
```
Comment (dnil) [inline scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html:240]: 👍

**Review by dnil — APPROVED** [review]
Great, that works!

**Pull request merged**